docker-compose.yml

```diff
@@ -25,7 +25,8 @@ services:
       - SPRING_DATASOURCE_PASSWORD=test
       - SPRING_JPA_PROPERTIES_HIBERNATE_DIALECT=org.hibernate.dialect.PostgreSQLDialect
       - SPRING_JPA_HIBERNATE_DDLAUTO=update
-      - LDES_STREAMS_0_=https://data.vlaanderen.be/ns/mobiliteit#Mobiliteitshinder
+      - LDES_STREAMS_0_MEMBER_TYPE=https://data.vlaanderen.be/ns/mobiliteit#Mobiliteitshinder
```

Comment:
Could you give context where this property is used? Since in the LDES Server we try to avoid member type in favor of the shacl targetClass property